README.md

```diff
@@ -29,19 +29,17 @@ The following pseudocode demonstrates running inference:
     Criteria<Image, Classifications> criteria =
             Criteria.builder()
                     .optApplication(Application.CV.OBJECT_DETECTION) // find object dection model
-                    .setTypes(Image.class, Classifications.class) // define input and output
-                    .optFilter("backbone", "resnet50") // choose network architecture
+                    .setTypes(Image.class, Classifications.class)    // define input and output
+                    .optFilter("backbone", "resnet50")               // choose network architecture
                     .build();
 
-    try (ZooModel<Image, Classifications> model = criteria.loadModel()) {
-        try (Predictor<Image, Classifications> predictor = model.newPredictor()) {
-            Image img = ImageFactory.getInstance().fromUrl("http://..."); // read image
-            Classifications result = predictor.predict(img);
+    ZooModel<Image, Classifications> model = criteria.loadModel();
```

Comment:
Why we remove try-catch?

It's better show real code here. We could improve it like:

```
    Image img = ImageFactory.getInstance().fromUrl("http://..."); // read image
    try (ZooModel<Image, Classifications> model = criteria.loadModel();
         Predictor<Image, Classifications> predictor = model.newPredictor()) {
         Classifications result = predictor.predict(img);
         // get the classification and probability
         ...
    }
```